kubetest/e2e.go

```diff
@@ -37,6 +37,9 @@ func run(deploy deployer, o options) error {
 	os.Setenv("KUBE_CONFIG_FILE", "config-test.sh")
 	// force having batch/v2alpha1 always on for e2e tests
 	os.Setenv("KUBE_RUNTIME_CONFIG", "batch/v2alpha1=true")
+	// force having apps/v1beta2 always on for e2e tests
+	// TODO: remove it once we enable apps/v1beta2 by default
+	os.Setenv("KUBE_RUNTIME_CONFIG", "apps/v1beta2=true")
```

Comment:
a) looks like this invalidates L39
b) how is this used? New code should follow the pop := pushEnv(); defer pop() pattern used elsewhere to explicitly set this where it is used and revert the change after the command that needs it is finished.